cogs/utility.py

```diff
@@ -0,0 +1,34 @@
+from discord.ext import commands, bridge
+import random
+import discord
+
+
+class utility(commands.Cog, name="utility"):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @bridge.bridge_command(brief="Generate a sona!")
+    async def sonagen(self, ctx):
+        primary_color = random.randint(0x000000, 0xFFFFFF)
+        color = random.choice(["Red", "Green", "Blue", "Pink", "Purple", "Brown", "Black", "White", "Orange", "Teal", "Light Green", "Light Blue", "Grey", "Yellow"])
+        species = random.choice(["Dragon", "Fox", "Deer", "Wolf", "Dog", "Bunny", "Protogen", "Hyena", "Tiger", "Lion", "Bird", "Otter", "Snake", "Cat", "Sergal", "Horse", "Shark", "Lizard"])
+        sonatype = random.choice(["Feral", "Anthro"])
+        sex = random.choice(["Male", "Male", "Male", "Male", "Female", "Female", "Female", "Female", "Hermaphrodite"])
+
+        embed=discord.Embed(title="Your Sona:", color = primary_color)
```

Comment:
Please adjust this. PEP8 conventions require that no spaces are provided around "equals" signs, as per convention E251.